scripts/ci-e2e.sh

```diff
@@ -68,10 +93,37 @@ prerequisites_powervs(){
     # Assigning PowerVS variables
     export IBMPOWERVS_IMAGE_NAME=${IBMPOWERVS_IMAGE_NAME:-"capibm-powervs-centos-streams8-1-22-4"}
     export IBMPOWERVS_SERVICE_INSTANCE_ID=${IBMPOWERVS_SERVICE_INSTANCE_ID:-"0f28d13a-6e33-4d86-b6d7-a9b46ff7659e"}
-    export IBMPOWERVS_NETWORK_NAME=${IBMPOWERVS_NETWORK_NAME:-"capi-e2e-test"}
+    export IBMPOWERVS_NETWORK_NAME=${BOSKOS_RESOURCE_NAME}
```

Comment:
what will happen if user doesn't use BOSKOS_RESOURCE_NAME? do we have anywhere default set for this var?

Reply:
yes, better to set a default value.